internal/metrics/metrics.go

```diff
@@ -56,12 +56,14 @@ const (
 	podInformerDelay                          = "foundry.spark.scheduler.informer.delay"
 	schedulingWaste                           = "foundry.spark.scheduler.scheduling.waste"
 	schedulingWastePerInstanceGroup           = "foundry.spark.scheduler.scheduling.wasteperinstancegroup"
+	driverCollocation                         = "foundry.spark.scheduler.scheduling.drivercollocation"
```

Comment:
nit: `driverexecutorinitialcollocation`? I want to convey that's the state when Spark application starts, even though it might change later.